src/admin/admin.service.ts

```diff
@@ -31,7 +31,6 @@ export class AdminService {
 
       const studentDatas = new Student(); // w przyszłości do usunięcia
 
-      studentDatas.role = UserRole.student;
```

Comment:
Why should we remove role?

Reply:
Nie jest do niczego potrzebna. 
To miałoby sens, gdyby Admin, Student i Recruter byli zbudowani na podstawie jakiejś bazowej encji/modułu, np. User.
My nic takiego nie mamy. Wszystko jest rozdzielone.